perl6-pod.html.markdown

```diff
@@ -0,0 +1,620 @@
+---
+language: Pod
+name: Perl 6 Pod
+contributors:
+    - ["Luis F. Uceta", "https://uzluisf.gitlab.io/"]
+filename: learnpod.pod6
+---
+
+Perl 6 Pod is an easy-to-use and purely descriptive mark-up language, 
+with no presentational components. Besides its use for documenting
+Perl 6 programs and modules, Pod can be utilized to write language
+documentation, blogs, and other types of document composition as well.
+
+Pod documents can be easily converted to HTML and many other formats
+(e.g., Markdown, Latex, plain text, etc.) by using the corresponding
+variant of the `Pod::To` modules (e.g. `<Pod::To::HTML>` for HTML conversion).
+
+Note: This document can be also be found as a Pod document
+[here](https://git.io/fpxKI).
+
+- [General Info](#general-info)
+- [Pod Basics](#pod-basics)
+	- [Basic Text Formatting](#basic-text-formatting)
+	- [Headings](#headings)
+	- [Ordinary Paragraphs](#ordinary-paragraphs)
+	- [Lists](#lists)
+	- [Code Blocks](#code-blocks)
+	- [Comments](#comments)
+	- [Links](#links)
+	- [Tables](#tables)
+- [Block Structures](#block-structures)
+	- [Abbreviated Blocks](#abbreviated-blocks)
+	- [Delimited Blocks](#delimited-blocks)
+	- [Paragraph Blocks](#paragraph-blocks)
+- [Configuration Data](#configuration-data)
+	- [Standard Configuration Options](#standard-configuration-options)
+	- [Block Pre-configuration](#block-pre-configuration)
+- [Semantic Blocks](#semantic-blocks)
+- [Miscellaneous](#miscellaneous)
+	- [Notes](#notes)
+	- [Keyboard Input](#keyboard-input)
+	- [Terminal Output](#terminal-output)
+	- [Unicode](#unicode)
+- [Rendering Pod](#rendering-pod)
+- [Accessing Pod](#accessing-pod)
+
+## General Info
+
+Every Pod document has to begin with `=begin pod` and end with `=end pod`.
+Everything between these two delimiters will be processed and used to
+generate documentation.
+
+```md
```

Comment:
Wherever beginning a syntax highlighted structure, start with just ` ``` ` instead of ` ```md `. We use Linguist to highlight and it will recognize the code as Pod.